### DISCUSSION TITLE
Help with unknown Display

### DISCUSSION THREAD

**Discussion by ruchuf** [olikraus/u8g2#2391] [General]
I try to give an old internet radio a second life.  For that purpose I wish to use the original grey scale LCD with 128x64 pixel.
The display measures 64,9mm x 39,1mm with a view port of 56mmx28mm.
Here are some pictures.
![Display-1](https://github.com/olikraus/u8g2/assets/30465454/6881f0ea-ace3-4fb6-97cf-b3d6137445bf)
![Display-2](https://github.com/olikraus/u8g2/assets/30465454/977f511b-ccc6-42fd-a9f0-d19096fdc0fc)
![Display-3](https://github.com/olikraus/u8g2/assets/30465454/9b83e6cf-222d-405f-8758-98282c534063)
![dual-IR-6S](https://github.com/olikraus/u8g2/assets/30465454/31ae2d6b-41dc-4b13-a63f-ae6115680cfe)

The connection pads are labeled: RES, RS, CS, SLK, DATA, VD0, GND, LED-, LED+.
So I assume that is an 4-wire SPI Display. I tried all available initialisations for 4-wire SPI displays with the GraphicsTest example.

With most tests the display stays just black, some more the display changes to grey, while others showed static random pixels either on black or grey background, without any change during the test program.

Only one init produces a little bit more: U8G2_SSD0323_OS128064_2_4W_SW_SPI
Here I get a series of random pixels and lines with different grey scale backgrounds. This is the only init, where the display content changes and shows a kind of reaction to the running code.

Is there a way, how I can do some more tests, maybe with changing the the init sequence of the display?  However I have no background knowledge what I could do there and how to proceed. 

I found an display with the same size here: https://www.aliexpress.com/item/1005006160620381.html, but unfortunately this display does not fit into the case, as the pcb is vertically to big.  The display itself looks very similar, but the background LED connections and the flex-pcb connector are different. This display works with U8G2_ST7565_64128N_F_4W_SW_SPI.

Has anybody here some ideas, what I can do?

**Comment by Pieter204** [comment]
@ruchuf .
Misschien kan ik u helpen ik werk met een Arduino u3 en heb deze Display SBC 128X64 er op aangesloten  en die gaat goed , die zouw er normaal kunnen voor doorgaan voor wat gij zoekt , en met deze gaat dat heel goed.
Gr
Pieter204

**Comment by olikraus** [comment]
Difficult, but of course you can change the init sequence in the code itself to see wether something improves.
I also suggest to redo your tests with the "ContastTest.ino" example instead of the graphics test example.
